Tests/LinuxMain.swift

```diff
@@ -4,80 +4,8 @@
 import XCTest
 
 @testable import PointFreeTests; @testable import StyleguideTests;
-extension AuthTests {
```

Comment:
Linux turned off now?

Reply:
um whoops. dont know what happened here, looking into it

Reply:
ah, it was cause `TestCase` moved modules. fixed the stencil template to just look for `...TestCase` subclasses.